src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+
+type TodoInfoProps = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  const currentUser =
+    usersFromServer.find(user => user.id === todo.userId) || null;
+  const todoClass = todo.completed
+    ? 'TodoInfo TodoInfo--completed'
+    : 'TodoInfo';
+
+  return (
+    <article key={todo.id} className={todoClass} data-id={todo.id}>
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      {currentUser && <UserInfo user={currentUser} />}
```

Comment:
[REACT KNOWLEDGE] - It's better to not render the component if the property that you pass to the component has `null` or `undefined` value. In this case, you should handle the case where there is no user in the parent component.